src/components/MyPage/MoodBoard.tsx

```diff
@@ -35,6 +35,10 @@ const MoodBoard = () => {
     }
   }, [hovered]);
 
+  moodBoardList.sort((a: MoodBoardServerData, b: MoodBoardServerData): number => {
+    return a.id - b.id;
+  });
```

Comment:
```suggestion ts
moodBoardList.sort((a: MoodBoardServerData, b: MoodBoardServerData): number => a.id - b.id;
  );
```
중괄호 빼면 return 없어도 데지 아나여?!